azurerm/data_source_kubernetes_cluster.go

```diff
@@ -161,6 +161,39 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"network_profile": {
```

Comment:
can [we document this block/these new fields here](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/d/kubernetes_cluster.html.markdown#L52)? (I've had to link to the Blame since you can't link to a regular line in a Markdown file in Github ¯\_(ツ)_/¯)